string[] joint_names
float32 trajduration
trajectory_msgs/JointTrajectoryPoint goalPoint
---
